mcmc/mcmc_framework.py

```diff
@@ -97,15 +97,18 @@ def run_mcmc(
         # Count the cores in the processor
         # (-1 if this chip also has a coordinator)
         n_cores = chip.n_user_processors
+        if chip.ip_address is None:
+            n_cores -= FecDataView.get_all_monitor_cores()
+        else:
+            n_cores -= FecDataView.get_ethernet_monitor_cores()
         if (chip.x, chip.y) in coordinators:
-            n_cores -= 3  # coordinator and extra_monitor_support (2)
+            n_cores -= 3  # coordinator
```

Comment:
Should probably make this -1 since the coordinator is only 1 core, and the other 2 were the extra monitor ones.

Reply:
oops fixed.